components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/NimbusBuilder.kt

```diff
@@ -59,6 +58,26 @@ abstract class AbstractNimbusBuilder<T : NimbusInterface>(val context: Context)
      */
     var onCreateCallback: (T) -> Unit = {}
 
+    /**
+     * Optional callback to be called everytime experiments have been applied.
+     *
+     * Experiment recipes are usually fetched shortly after startup, and those pending recipes are
+     * applied at the following startup.
+     *
+     * This is not usually needed.
+     */
+    var onApplyCallback: () -> Unit = {}
+
+    /**
+     * Optional callback to be called everytime experiments have been fetched.
+     *
+     * Experiment recipes are usually fetched shortly after startup, and those pending recipes are
+     * applied at the following startup.
+     *
+     * This is not usually needed.
+     */
+    var onFetchCallback: () -> Unit = {}
+
```

Comment:
Knowing this was a breaking change, I took the time to refactor the NimbusBuilder for Android and pull in the fetchCallback and applyCallback so they work in the same for when in Android Components and just with the DefaultNimbusBuilder in Application Services.